fury-on-gke/manifests/logging/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../vendor/katalog/logging/cerebro
+  - ../../vendor/katalog/logging/logging-operator
+  - ../../vendor/katalog/logging/logging-operated
+  - ../../vendor/katalog/logging/configs
+  - ../../vendor/katalog/logging/opensearch-single
+  - ../../vendor/katalog/logging/opensearch-dashboards
```

Comment:
minio-ha is missing here, without it the logging stack does not work.
```suggestion
  - ../../vendor/katalog/logging/opensearch-dashboards
  - ../../vendor/katalog/logging/minio-ha
```
⚠️ remember to update also the README.md file

Reply:
done